test/OrderModelTests.js

```diff
@@ -24,45 +28,45 @@ describe('Order model and join table defination', function () {
       inventory: 100,
       country: 'Taiwan',
       imageUrl:
-                'https://images-na.ssl-images-amazon.com/images/I/51GkdanTqfL.jpg',
+        'https://images-na.ssl-images-amazon.com/images/I/51GkdanTqfL.jpg',
     });
     PineappleCake = await Product.create({
       title: 'Pineapple Cake',
       brand: 'Jun-Mei',
       description:
-                'A buttery, shortbread-like treat with a pineapple jam filling.',
+        'A buttery, shortbread-like treat with a pineapple jam filling.',
       price: 36.99,
       inventory: 139,
       country: 'Taiwan',
```

Comment:
Hi Kevin, in Product. create,  you would have to use "location" instead of "country"...I had changed it to avoid the naming collision ...